Docs: Rewrite preferences docs (#31154)
grafana/grafana: docs/sources/administration/preferences/change-grafana-theme.md

+++
title = "Change UI theme"
description = "How to set the Grafana UI theme"
keywords = ["grafana", "configuration", "documentation", "home"]
weight = 200
+++

# Change Grafana UI theme

> In Grafana, you can the UI them used in the Settings or Preferences. You can set the UI theme for the server, an organization, a team, or your personal user account. This topic provides instructions for each task.

Reviewer: ```suggestion
In Grafana, you can modify the UI theme configured in the Settings or Preferences. Set the UI theme for the server, an organization, a team, or your personal user account using the instructions in this topic.
```

{{< docs/shared "preferences/some-tasks-require-permissions.md" >}}

## Theme options

The theme affects how Grafana displays graphs, menus, other UI elements.

### Default

**Default** is either the dark theme or the theme selected in a higher level. For example, if an Organization administrator set the **Light** theme, then that is the default for all the teams in that organization.

### Dark

Here is an example of the dark theme.

![Dark theme example](/img/docs/preferences/dark-theme-7-4.png)

> ### Light

Author: Yes. There are more themes coming soon, so this section will expand. It is useful to be able to link directly to a particular theme.
